recipes/jsonextended/LICENSE.TXT

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019 Chris Sewell
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
```

Comment:
Future releases should include this file else the tarball on PyPI is in violation of this line. You can do this by adding it to your `MANIFEST.in` file.

When that is done you can delete this file and get it from the source tarball instead.

Reply:
will do @chrisburr, just to check, to create future releases, I need to fork the feedstock, change the sha and version then PR?